### DISCUSSION TITLE
Bref Dashboard: what do you need?

### DISCUSSION THREAD

**Discussion by mnapoli** [brefphp/bref#813] [Ideas]
Hey everyone,

I would like to improve the Bref Dashboard, I'm interested in your experience.

**What are you missing in your daily work?**

**What would be super useful to you?**

Reminder: you can run it with:

```bash
vendor/bin/bref dashboard
```

**Comment by danniehansen** [comment]
@mnapoli Not sure if it's already possible. Couldn't find it in docs or source code. But an option to load a different serverless configuration than `serverless.yml` would be really useful. I have 5 separate configs because i needed 5 separate API Gateways. So unfortunately I'm not able to use the dashboard now.

**Reply by mnapoli** [reply]
Ha, good news! I've been working exactly on that lately. It could be better, but there is now a "stack selector" that lets you choose your application if you don't use serverless.yml.

Start the dashboard manually like this:

```
docker pull bref/dashboard
docker run --rm -p 8000:8000 -v $HOME/.aws:/root/.aws:ro bref/dashboard
```

**Reply by danniehansen** [reply]
Really nice! Going to take it for a spin tomorrow. Thanks! 👍

**Reply by danniehansen** [reply]
That's awesome! Worked really well. A few initial thoughts after viewing it:

1. Cost doesn't appear to be right. 0$ for all of our functions.
2. (Suggestion) Ability to pan the graphs as well as hovering over the graph to see invocations at a specific time.
3. (Suggestion) Ability to view concurrency graph as well.
4. (Suggestion) Ability to see reserved concurrency on functions in the overview & maybe even a current concurrency counter? I always find it hard to locate where my concurrency reservations are & match it up with current concurrent usage to find out which of my functions is getting throttled (unless looking through all CloudWatch graphs.

**Reply by mnapoli** [reply]
Great suggestions, thanks a lot!

Regarding costs, just to be sure I understand: was it showing $0 but it should be more?

Or was it showing more, and it should be $0?

Because that "costs" widget counts the cost of the whole application, and ignores the free tier or credits.

**Reply by danniehansen** [reply]
No problem at all 👍 

It's showing 0$ but should be more.
